execute/row_fn.go

```diff
@@ -197,7 +197,7 @@ func (f *rowFn) eval(ctx context.Context, row int, cr flux.ColReader, extraParam
 		f.arg0.Set(col.Label, ValueForRow(cr, row, j))
 	}
 	for k, v := range extraParams {
-		f.arg0.Set(k, v)
+		f.args.Set(k, v)
```

Comment:
Can you remind me what this does again? Why the switch from arg to arg0?

Reply:
The previous code was wrong. It's too set the values on the arguments. I misunderstood it's purpose previously when fixing it and it's only used by reduce.